tests/test_single_topology_v3.py

```diff
@@ -206,7 +206,7 @@ def get_max_distance(x0):
 
         print("Checking", get_mol_name(mol_a), "->", get_mol_name(mol_b))
         mcs_threshold = 0.75  # distance threshold, in nanometers
```

Comment:
0.75 -> 2.0 for consistency?

Reply:
Modified all instances of `mcs_threshold = 0.75` to `mcs_threshold = 2.0` in https://github.com/proteneer/timemachine/pull/812/commits/2b91826c31d69235fdd90b5b9c3b31cc82533bf4